src/components/__tests__/response_error.test.tsx

```diff
@@ -6,14 +6,14 @@ import { shallow } from 'enzyme';
 import { I18n } from '../../utils/i18n';
 
 // component
-import ResponseError, { ErrorMessageTemplates, FieldTranslations } from '../response_error';
+import ResponseError from '../response_error';
 
-const ERROR_CODE_TO_I18NC_TEMPLATE: ErrorMessageTemplates = {
-    fieldError: I18n.translate('error message', '%{field_name} was invalid: %{title}.'),
+const ERROR_CODE_TO_I18NC_TEMPLATE = {
+    fieldError: () => I18n.translate('error message', '%{field_name} was invalid: %{title}.'),
 };
 
-const FIELD_NAME_TO_I18NC: FieldTranslations = {
-    username: I18n.translate('form field username', 'Username'),
+const FIELD_NAME_TO_I18NC = {
```

Comment:
Now we have no type checking here and in commons

Reply:
There's type checking on the function itself though, this is only the test input. If it's wrong, your editor will indicate that the type doesn't match the expected type.